app/screens/home/search/results/empty_files.tsx

```diff
@@ -0,0 +1,71 @@
+                fill='#BABEC9'
+            />
+            <Path
+                opacity='0.4'
+                d='M102.407 37.508C97.653 32.6931 91.6526 30.2132 84.4183 30.0801C77.317 30.2132 71.3529 32.6811 66.526 37.508C61.7112 42.3349 59.2312 48.299 59.0981 55.4003C59.2312 62.6346 61.6991 68.6229 66.526 73.3893C71.3529 78.1437 77.317 80.6599 84.4183 80.9019C91.6526 80.6478 97.6409 78.1437 102.407 73.3893C107.162 68.6229 109.666 62.6225 109.92 55.3882C109.666 48.2869 107.162 42.3228 102.407 37.508Z'
+                fill='white'
+            />
+            <Path
+                d='M61.1129 50.8816C63.2645 43.89 67.3379 38.6565 73.3391 35.1917C79.3403 31.727 85.9093 30.816 93.0401 32.4484C96.9394 33.4443 100.337 35.1659 103.26 37.6111C99.9912 34.044 95.8452 31.5739 90.8322 30.1948C83.7015 28.5624 77.1001 29.492 71.0449 32.9879C64.9898 36.4839 60.884 41.7361 58.7324 48.7277C57.4095 53.7648 57.4756 58.5904 58.9414 63.1982C60.3965 67.8123 63.0084 71.6882 66.7819 74.8091C63.9488 71.7216 61.9993 68.0958 60.9383 63.9149C59.888 59.7278 59.9467 55.3925 61.1129 50.8816Z'
+                fill='black'
+                fillOpacity='0.4'
+            />
+            <Path
+                d='M53.24 53.4287C53.7476 45.9235 56.7935 39.3323 62.3654 33.6553C68.7108 27.6054 76.0353 24.5745 84.3388 24.5745C92.6423 24.5745 99.9547 27.6054 106.3 33.6553C112.38 39.9698 115.426 47.2465 115.426 55.5095C115.426 63.7725 112.38 71.0612 106.3 77.3637C100.474 82.9204 93.8147 85.9153 86.3331 86.3483C78.8515 86.7933 71.9501 84.7366 65.6046 80.2022L60.8546 84.929L54.775 78.8791L59.525 74.1523C54.8354 67.8498 52.7445 60.9339 53.24 53.4287ZM66.366 73.2983C71.1886 78.0252 77.1473 80.5269 84.2421 80.7675C91.4699 80.5149 97.4528 78.0252 102.215 73.2983C106.965 68.5715 109.467 62.6058 109.721 55.4133C109.467 48.3531 106.965 42.4234 102.215 37.6244C97.4649 32.8374 91.4699 30.3718 84.2421 30.2395C77.1473 30.3718 71.1886 32.8254 66.366 37.6244C61.5556 42.4234 59.0778 48.3531 58.9449 55.4133C59.0778 62.6058 61.5435 68.5715 66.366 73.2983Z'
+                fill='#BABEC9'
+            />
+            <Path
+                d='M33.7979 113.687C35.3653 114.135 36.5259 113.978 37.2797 113.215L61.9397 86.1514C62.6935 85.3885 63.0045 84.4077 62.8849 83.1968C62.7652 81.9859 62.0114 80.6296 60.6235 79.1039C59.1159 77.7114 57.7758 76.9122 56.5793 76.7184C55.3828 76.5247 54.4136 76.8758 53.6598 77.7719L26.7385 102.729C25.9847 103.492 25.8651 104.63 26.3676 106.156C26.8701 107.681 27.8154 109.207 29.1913 110.733C30.687 112.259 32.2305 113.239 33.7979 113.687Z'
+                fill='#FFBC1F'
+            />
+            <Path
+                d='M31.9929 97.8425L42.0738 108.007L54.2274 94.6662L45.3132 85.5606L31.9929 97.8425Z'
+                fill='#7A5600'
+            />
+            {/* <defs> */}
```

Comment:
I am not sure what this portion of the SVG references in the image.  I didn't see any difference when it was added or omitted.

Reply:
Yep, I think this can be omitted